iphone/Classes/TiUIListView.m

```diff
@@ -2046,17 +2047,12 @@ - (void)willDismissSearchController:(UISearchController *)searchController
              [_searchWrapper refreshView:nil];
         }
     }
-    //IOS7 DP3. TableView seems to be adding the searchView to
-    //tableView. Bug on IOS7?
-    [self clearSearchController:self];
+    [searchViewProxy ensureSearchBarHeirarchy];
     [_tableView reloadData];
 }
 
 - (void)presentSearchController:(UISearchController *)controller
 {
-    id dimBgValue = [[self  proxy] valueForKey:@"dimsBackground"];
-    searchController.dimsBackgroundDuringPresentation = [TiUtils boolValue:dimBgValue def:NO];
```

Comment:
This property is of no use, until searchResultController is passed in UISearchViewController. So removed it and put a default value while initializing searchViewController.